unpackaged/config/rd2_post_config/objectTranslations/npe03__Recurring_Donation__c-de.objectTranslation

```diff
@@ -1,26 +1,263 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
-    <fields>
-        <help>Wählen Sie bei einer Wiederkehrenden Spende mit offenem Ende &quot;Offen&quot;. Wählen Sie &quot;Keine&quot; bei einer Wiederkehrenden Spende mit fester Dauer. Wählen Sie &quot;Geschlossen&quot;, um eine wiederkehrende Spende zu schließen.</help>
-        <label>Deprecated-Status &quot;Unbefristet&quot;</label>
-        <name>npe03__Open_Ended_Status__c</name>
-    </fields>
-    <fields>
-        <help>Gibt Salesforce an, wie der Betrag für die einzelnen untergeordneten Opportunities auf der Grundlage dessen, was Sie in das Betragsfeld eingegeben haben, berechnet werden soll. Nur für Wiederkehrende Spenden mit fester Dauer.</help>
-        <label>Deprecated-Zeitplantyp</label>
```

Comment:
@GloryChoe The translations for the Recurring Donations object in the /unpackaged/config/rd2_post_config folder are extremely close. There's only one minor change needed, though it's possible I need to do it manually in each of the translation files. 

There are three fields where I added the word "Deprecated-" before the field label in the translation file(s): `npe03__Open_Ended_Status__c`, `npe03__Schedule_Type__c` and `npe03__Total__c`. The word "Deprecated-" is not in the object file in the unpackaged folder for these fields due to technical reasons that prevent the label from being modified in the object file. If I want to include "Deprecated" on these fields for each language, I think the only way to effectively do it is to manually edit the label on each field in each language file. That's not a big deal for me to do (considering it was done previously). Do you think it's possible to get the translation for "Deprecated-" for each of the languages for me to use with these fields?

Reply:
OK! I'll request translations for "Deprecated-" for all languages. @force2b Please allow a couple of days for the turnaround.

Reply:
Thank you @GloryChoe. Very much appreciated.

Reply:
Hi @force2b Will it be easier if I ask the linguists to add "Deprecated-" in each of the labels for these 3 files instead since I'm sending the project back to them for "Recurring Donations" translation added from relationshipLabel? I'm worried that if we don't implement this update on our translation tool side, the "deprecated - " tags you add will be removed next time I deliver loc updates from the translation tool.

Reply:
Hi @force2b Here are the translations for "Deprecated-"

German: Ersetzt-

Spanish: Desusado-

French: Déprécié-

Japanese: 廃止ー

Dutch: Verouderd-

Reply:
@force2b FYI I think ":" might be a better way as "hyphens" can mean something else in some of the languages. My proposal is:

German: Ersetzt: 

Spanish: Desusado: 

French: Déprécié : 

Japanese: 廃止：

Dutch: Verouderd:

Reply:
Thank you @GloryChoe. I updated the unpackaged translations with these prefixes.